docs/source/shared.rst

```diff
@@ -101,7 +101,10 @@ overhead grows linearly.  The asymptotic cost per task depends on the scheduler.
 The schedulers that depend on some sort of asynchronous pool have costs of a few
 milliseconds and the single threaded schedulers have costs of a few microseconds.
 
-.. image:: images/scaling-nodes.png
+.. figure:: images/scaling-nodes.png
+   :alt: Duration in seconds on the y-axis versus number of edges per task on the x-axis. The time to schedule the entire graph is constant initially, followed by a linear increase after roughly 500 tasks for multiprocessing and threaded schedulers and 10 tasks for async and core schedulers. The inverse is true for the cost per task, with a linear cost decrease, followed by more or less constant cost.
```

Comment:
```suggestion
   :alt: Graph depicting how well Dask scales with the number of nodes in the task graph. Graph shows the duration in seconds on the y-axis versus number of edges per task on the x-axis. The time to schedule the entire graph is constant initially, followed by a linear increase after roughly 500 tasks for multiprocessing and threaded schedulers and 10 tasks for async and core schedulers. The inverse is true for the cost per task, with a linear cost decrease, followed by more or less constant cost.
```

Good to start with a sentence describing the whole image, before diving in to the details.